src/backend/src/main/kotlin/au/com/cding21/security/encryption/RSAServiceImpl.kt

```diff
@@ -0,0 +1,67 @@
+package au.com.cding21.security.encryption
+
+import org.bouncycastle.crypto.digests.SHA512Digest
+import org.bouncycastle.crypto.engines.RSAEngine
+import org.bouncycastle.crypto.signers.RSADigestSigner
+import org.bouncycastle.crypto.util.OpenSSHPrivateKeyUtil
+import org.bouncycastle.crypto.util.OpenSSHPublicKeyUtil
+import org.bouncycastle.util.io.pem.PemReader
+import java.io.StringReader
+import java.security.SignatureException
+import java.util.Base64
+
+class RSAServiceImpl(private val pubKey: String, private val privateKey: String): AsymmetricEncryptionService {
+    override fun getPubKey(): String {
+        return pubKey
+    }
+
+    override fun encrypt(plaintext: String, foreignPubKey: String): Pair<ByteArray, ByteArray> {
+        var formattedPrivateKey = privateKey.replace(" ", "\n")
```

Comment:
Temp solution to deal with badly formatted env variables. Fine for now as it isn't used anywhere but will need better solution moving forward